core/src/main/python/validate.py

```diff
@@ -188,7 +189,7 @@ def main(args):
                     exit_code = ExitCode.WARNING
 
     except ValidateException, ve:
-        exit_code = ExitCode.ERROR
+        exit_code = CommandLineArgUtil.PROG_ERROR_EXIT_CODE
```

Comment:
Why does this need to be changed?

Reply:
This may have been a bad merge.  I didn't intentionally change it.